bolt/lib/Passes/SplitFunctions.cpp

```diff
@@ -715,6 +715,12 @@ Error SplitFunctions::runOnFunctions(BinaryContext &BC) {
   if (!opts::SplitFunctions)
     return Error::success();
 
+  if (BC.IsLinuxKernel && BC.BOLTReserved.empty()) {
+    BC.errs() << "BOLT-ERROR: split functions require reserved space in the "
```

Comment:
longer term idea/ drive-by comment: having to remember to type `BOLT-ERROR:` every time is cognitive load we might want to avoid. Potentially we might want at some point to do a global mark-and-sweep and introduce something like `llvm::bolt::bolt_error()`, or something along these lines.

Reply:
Totally. Patches are welcome :)